src/data/faq_de.json

```diff
@@ -65,11 +194,28 @@
                         "indexed": true,
                         "accordion": [
                             {
-                                "title": "Hier bereiten wir Antworten zu Fragen vor, welche Möglichkeiten es nach Ende der Entwicklungsarbeiten an der CWA künftig noch gibt.",
-                                "anchor": "ramp_down_alternatives",
+                                "title": "Was passiert mit den CovPass-Apps? Werden sie auch eingestellt? Wenn ja, wann?",
+                                "anchor": "ramp_down_covpass",
+                                "active": false,
+                                "textblock": [
+                                    "Die CovPass-Apps werden in Übereinstimmung mit der Geltungsdauer der europarechtlichen Rechtsgrundlagen weiter betrieben. Die hierfür maßgebliche Verordnung (EU) 2021/953 gilt derzeit bis zum 30. Juni 2023."
+                                ]
+                            },
+                            {
+                                "title": "Kann ich meine Daten aus der App bekommen? Gibt es ein Backup?",
+                                "anchor": "ramp_down_backup",
+                                "active": false,
+                                "textblock": [
+                                    "Ja, zum Backup der Zertifikate nutzen Sie bitte die Funktionen Einzel-Export (auf der Detail-Ansicht des Zertifikats) oder Zertifikate exportieren für den Gruppenexport aller Zertifikate in eine PDF-Datei. Eine Anleitung zum Export von Zertifikaten finden Sie hier im FAQ: <a href='#eu_dcc_export' target='_blank'>Wie erstelle ich einen Ausdruck von digitalen COVID-Zertifikaten der EU?</a>",
+                                    "Zur Sicherung der Daten aus dem Kontakt-Tagebuch können Sie die Funktion Einträge exportieren verwenden. Weitere Information dazu finden Sie im <a href='../../blog/2020-12-28-corona-warn-app-version-1-10/' target='_blank'>Blog-Artikel zum Kontakt-Tagebuch</a>"

```

Comment:
Put Einträge exportieren in quotes like in the English version.

Reply:
Updated in https://github.com/corona-warn-app/cwa-website/pull/3435/commits/73c82ad32454368aa9de62973567f834a1c8e0ca, thanks!